datalad/utils.py

```diff
@@ -1539,7 +1539,7 @@ def make_tempfile(content=None, wrapped=None, **tkwargs):
 
     filename = {False: tempfile.mktemp,
                 True: tempfile.mkdtemp}[mkdir](**tkwargs_)
-    filename = realpath(filename)
+    #filename = realpath(filename)
```

Comment:
might be worth checking why it was added - there was a reason I guess for realpath, if not -- remove and leave a comment

Reply:
I will take this out (was committed unintentionally). I carry this change locally, as it makes it possible for the test battery to discover symlink issue. But I have no juice to figure them all out in -core (all that code is replaced in -revolution.

Reply:
Force-pushed removal.